pytest_mh/utils/fs.py

```diff
@@ -611,3 +611,27 @@ def chmod(self, mode: str, path: str, args: list[str] = []) -> SSHProcessResult:
         self.logger.info(f'Changing mode bits for "{path}" on {self.host.hostname}')
 
         return self.host.ssh.exec(["chmod", *args, mode, path])
+
+    def chown(
+        self, path: str, user: str | None = None, group: str | None = None, args: list[str] = []
+    ) -> SSHProcessResult:
+        """
+        Change file owner and group.
+
+        :param path: Path to file
+        :type path: str
+        :param user: New file owner, if None then user remains same, defaults to None
+        :type user: str | None, optional
+        :param group: New file group, if None then group remains same, defaults to None
+        :type group: str | None, optional
+        :param args: Additional options, defaults to []
+        :type args: list[str], optional
+        :return: Result of process
+        :rtype: SSHProcessResult
+        """
+        self.backup(path)
+        self.logger.info(f'Changing owner for "{path}" on {self.host.hostname}')
+
+        mode = f"{user}" if user else ""
+        mode += f":{group}" if group else ""
+        return self.host.ssh.exec(["chown", mode, *path.split(), *args])
```

Comment:
Please, use `log_level=SSHLog.Error` so the command is logged only on failure. This will reduce the amount of text in the log output. Please, do the same for `wc` and `chmod` (different commit).